src/guiapi/include/IWindowMenuItem.hpp

```diff
@@ -39,8 +39,7 @@ class IWindowMenuItem {
     void Enable() { enabled = true; }
     void Disable() { enabled = false; }
     bool IsEnabled() const { return enabled; }
-    void SetHidden() { hidden = true; }
```

Comment:
please try to avoid bool parameters in methods, 
I agree original naming is terrible, consider
```
    void Hide() { hidden = true; }
    void Show() { hidden = false; }
```
instead